mne_bids/write.py

```diff
@@ -1887,11 +1888,81 @@ def write_anat(image, bids_path, landmarks=None, deface=False,
     return bids_path
 
 
+@deprecated(extra='mark_bad_channels is deprecated in favor of mark_channels '
+                  'and will be removed in v0.10.')
 @verbose
 def mark_bad_channels(ch_names, descriptions=None, *, bids_path,
                       overwrite=False, verbose=None):
     """Update which channels are marked as "bad" in an existing BIDS dataset.
 
+    This modifies entries in the ``channels.tsv`` file in a BIDS dataset.
+
+    Parameters
+    ----------
+    ch_names : str | list of str
+        The names of the channel(s) to mark as bad. Pass an empty list in
+        combination with ``overwrite=True`` to mark all channels as good.
+    descriptions : None | str | list of str
+        Descriptions of the reasons that lead to the exclusion of the
+        channel(s). If a list, it must match the length of ``ch_names``.
+        If ``None``, no descriptions are added.
+    bids_path : mne_bids.BIDSPath
+        The recording to update. The :class:`mne_bids.BIDSPath` instance passed
+        here **must** have the ``.root`` attribute set. The ``.datatype``
+        attribute **may** be set. If ``.datatype`` is not set and only one data
+        type (e.g., only EEG or MEG data) is present in the dataset, it will be
+        selected automatically.
+    status : str | list of str
+        The status of the channels ('good', or 'bad'). Default is 'bad'. If it
+        is a list, then must be a list of 'good', or 'bad' that has the same
+        length as ``ch_names``.
+    overwrite : bool
+        If ``False``, only update the information of the channels passed via
+        ``ch_names``, and leave the rest untouched. If ``True``, update the
+        information of **all** channels: mark the channels passed via
+        ``ch_names`` as bad, and all remaining channels as good, also
```

Comment:
This needs to be updated

Reply:
I propose to just remove the `overwrite` functionality. It seems somewhat redundant and confusing. Users should rather like you said just explicitly pass 'good', or 'bad' to the channel names they want to reset. If they really want to reset all channels to good, it's also easy:

`mark_channels(bids_path, raw.ch_names, 'good')`

Reply:
Is there a clean way to deprecate a kwarg in `mark_channels`? i.e. `overwrite`?